nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml

```diff
@@ -127,8 +133,113 @@
             <version>5.6.8</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-avro-record-utils</artifactId>
+            <version>1.7.0-SNAPSHOT</version>
```

Comment:
This one should be 1.8.0-SNAPSHOT now, sorry it's taken so long to get through

Reply:
Done.